_sass/custom.scss

```diff
@@ -11,39 +11,3 @@
         content: url("/img/logo-dark.jpg")
     }
 }
-
-// Admonition styles
-.admonition {
```

Comment:
If you are ever adding custom CSS like this, you're probably doing it wrong. ☺️ 

Bootstrap has a component for this:
https://getbootstrap.com/docs/5.3/components/alerts/